interface/billing/billing_tracker.php

```diff
@@ -104,8 +104,8 @@
                             return data;
                         }
                     },
-                    { 
-                        "data": "created_at", 
+                    {
```

Comment:
php cbf flagged these lines, I didn't really modify them but it auto-formatted them.